daemon/mgr/cri_utils.go

```diff
@@ -365,3 +367,24 @@ func imageToCriImage(image *apitypes.ImageInfo) (*runtime.Image, error) {
 		Size_:       size,
 	}, nil
 }
+
+// ensureSandboxImageExists pulls the image when it's not present.
+func (c *CriManager) ensureSandboxImageExists(ctx context.Context, image string) error {
+	_, err := c.ImageMgr.GetImage(ctx, image)
+	// TODO: maybe we should distinguish NotFound error with others.
```

Comment:
Yes, only NotFound error shall we pull the image, right? @YaoZengzeng

Reply:
@allencloud  yes 😄